### PR TITLE
Fix some grammar issues, and some stylistic suggestions.

### PR DESCRIPTION
**Pull request by maier49**
**Type:** Words

The following has been addressed in the PR:

* [ ] There is a related issue
* [ ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Fixes some typos and grammatical issues, and also just has some suggestions for alternative wordings.

### CODE REVIEW

File: README.md
Comment (dylans) [inline README.md]: while we're making these changes, place holder -> placeholder

File: README.md
Comment (dylans) [inline README.md]: It only -> The harness only

**Review by dylans — APPROVED** [review]
Changes look great. I saw two more things in the review that would be great to address now as well if you have time, but that doesn't block approval.

**Comment by maier49** [comment]
I made the changes requested. I'm thinking it would be better to wait on @lzhoucs 's PR to get merged since this has a lot more changes.

**Comment by dylans** [comment]
@maier49 ok, I've landed the other PR, so this one needs a quick rebase and then we can land it as well.

**Pull request merged**
